TRACK_PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+Thank you for this PR! :tada:
+
+When we review your submission we will look for the following.
+
+- [ ] Trailing whitespace and extraneous blank lines 
+- [ ] Indentation and code formatting issues
+- [ ] Spelling and grammatical errors
+- [ ] Code style
+- [ ] Use of non-standard libraries
+- [ ] Extraneous config files
+- [ ] Deviations from the specifications set out for the exercise in [x-common](https://github.com/exercism/x-common/tree/master/exercises)
```

Comment:
I think it would alert that deviances will tend to be noted, just like extraneous config files or non-standard libraries would be.